appenders/python/pulse/formatters.py

```diff
@@ -26,5 +27,11 @@ def format(self, record):
         data["level"] = record.levelname
         data["message"] = record.msg
         data["threadName"] = record.threadName
+        try:
+            data["hostname"] = socket.gethostname()
+            if not data["hostname"]:
+                raise RuntimeError('No HostName received via socket')
+        except Exception:
+            logging.error('Failed.', exc_info=True)
```

Comment:
This should be swallowed. If we can't get a hostname it's not critical. If we log the message we could end up in an infinite loop.

Reply:
make sense.